### DISCUSSION TITLE
We should create a community Discord server and close Slack server.

### DISCUSSION THREAD

**Discussion by devlablt** [symfony/symfony#51099] [Q&A]
### Description

Instead of using Slack, we should consider creating a Discord server and migrate our entire Slack community there.

One of the significant issues with Slack is that we are using the free version, which automatically "deletes" all messages after 90 days. This means that if someone encounters a problem that you're facing now, they won't be able to search directly on Slack to find the past discussions on that topic.

Many other frameworks exclusively use Discord for their community, such as:

[Laravel](https://laravel.com/) - You can find the Discord icon if you scroll down to the bottom.
[Next.js](https://nextjs.org/blog/discord)

Therefore, I suggest we create a Discord server and prominently feature it on the main page of https://symfony.com/.

**Comment by fancyweb** [comment]
Hello @devlablt I converted your issue to a discussion since it's unrelated to Symfony's code (not a bug or a feature).

**Comment by asispts** [comment]
Agreed. Even we don't have any unofficial channels :worried: 

<img src="https://github.com/symfony/symfony/assets/79239132/76b79de1-0dcf-42f1-a161-4eda52417b20" width="480"/>

**Comment by Seb33300** [comment]
Bootstrap is also using discord: https://github.com/twbs/bootstrap#community

More and more communities are switching to discord.
On discord the users can boost a server to unlock paid features while we are stuck on the free version of Slack.
